flow.json

```diff
@@ -1,18 +1,39 @@
 {
-  "emulators": {
-    "default": {
-      "port": 3569,
-      "serviceAccount": "emulator-account"
-    }
-  },
   "contracts": {
-    "FooContract": "./contracts/FooContract.cdc",
-    "ArrayUtils": "./contracts/ArrayUtils.cdc",
-    "StringUtils": "./contracts/StringUtils.cdc",
-    "ApprovalVoting": "./contracts/ApprovalVoting.cdc"
+    "ApprovalVoting": {
+      "source": "contracts/ApprovalVoting.cdc",
+      "aliases": {
+        "testing": "0x0000000000000007"
```

Comment:
This is interesting! Curious about how we determine the testing address and how the testing alias is used

Reply:
Hey @sisyphusSmiling :wave: 
Basically, I borrowed an idea from Flow Playground, where we use simple, sequential addresses to be easier to deal with. The testing framework now offers a pool a 10 predefined accounts, where users can deploy their contracts under testing.

```cadence
import Test
import BlockchainHelpers
import "ApprovalVoting"

access(all) let admin = Test.getAccount(0x0000000000000007)

access(all)
fun testInitializeEmptyProposals() {
    let txResult = executeTransaction(
        "../transactions/initialize_proposals.cdc",
        [proposals],
        admin
    )
}
```

Developers can also gain access to this account, in order to use it for signing transactions.
Stay tuned as I am going to update all relevant projects with the new API, including `HybridCustody`